modules/Bio/EnsEMBL/Compara/RunnableDB/HomologyAnnotation/DiamondBlastp.pm

```diff
@@ -0,0 +1,100 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 NAME
+
+Bio::EnsEMBL::Compara::RunnableDB::HomologyAnnotation::DiamondBlastp
+
+=head1 DESCRIPTION
+
+Create fasta file containing batch_size number of sequences. Run DIAMOND and parse the output into
+PeptideAlignFeature objects. Store PeptideAlignFeature objects in the compara database
+
+=cut
+
+package Bio::EnsEMBL::Compara::RunnableDB::HomologyAnnotation::DiamondBlastp;
+
+use strict;
+use warnings;
+
+use base ('Bio::EnsEMBL::Compara::RunnableDB::BlastAndParsePAF');
+
+sub get_queries {
+    my $self = shift @_;
+
+    my $start_member_id = $self->param_required('start_member_id');
+    my $end_member_id   = $self->param_required('end_member_id');
+
+    #Get list of members and sequences
+    my $member_ids = $self->compara_dba->get_HMMAnnotAdaptor->fetch_all_seqs_missing_annot_by_range($start_member_id, $end_member_id, 'no_null');
+    return $self->compara_dba->get_SeqMemberAdaptor->fetch_all_by_dbID_list($member_ids);
+}
+
+sub run {
+    my $self = shift @_;
+
+    #my $diamond_exe             = $self->param('diamond_exe'); 
```

Comment:
```suggestion
```

Reply:
I have left this to go with the below comment, it **stands out** more as an _**in your face**_ reminder that it needs changing as soon as we have a farm-wide executable to stick somewhere